tests/tests/tier0/bluechi-get-default-target/test_bluechi_agent_get_default_target.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+
+from bluechi_test.config import BluechiAgentConfig, BluechiControllerConfig
+from bluechi_test.machine import BluechiAgentMachine, BluechiControllerMachine
+from bluechi_test.test import BluechiTest
+
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.exec_run("systemctl get-default")
+    assert systemd_result in bc_result
+
+    node_foo.exec_run("systemctl set-default multi-user.target")
```

Comment:
Do we need to run `set-default` as a part of the test? Isn't it enough for testing get-default just to compare `bluechictl` and `systemctl` results?

Reply:
I am not sure about this one. I feel like I need to see that if the default target changes the get default command will get the correct answer. But if you think it's unnecessary I will delete it.

Reply:
Well, for me checking get is enough.
But on the other hand we can even have a single test for both get and set:

1. Run both bluechictl and systemctl get and compare values
2. Run bluechictl set
3. Run both bluechictl and systemctl get and check if the new target is really reported by both

Having a test like this would verify both functions

Reply:
I thought about this test. But if this fails I am not sure we can know if it is a get or a set problem. Or maybe I am wrong?

Reply:
For step 1. verifies get functionality (bluechictl and systemctl should return the same value). 

Steps 2. and 3. verified set functionality. Step 2 operation should finish with success and then in step 3 we verifed that requested target was really applied and reported the same by both bluchictl and systemctl.

If something fails, then of course we would need to go to logs to find out which operation failed. But again we can have either a single test for both or specialized tests for each operation, I don't have a strong preference here